src/python/doc/index.rst

```diff
@@ -37,16 +45,10 @@ Cover complexes
 
 .. include:: nerve_gic_complex_sum.inc
 
-Data structures and basic operations
-************************************
-
-Data structures
-===============
-
-Simplex tree
-------------
+Tangential complex
+==================
 
-.. include:: simplex_tree_sum.inc
+.. include:: tangential_complex_sum.inc
```

Comment:
It's ok for me to change the modules list, but we have to move the C++ one accordingly and also the [web site introduction](https://gudhi.inria.fr/introduction/).